src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
+        Union[
+            KubernetesKedaBackend,
+            OrchestrationInterfacesBackend,
+            OCMBackend,
+            LocalBackend,
+            MockupBackend,
+        ]
     ] = Field(
-        {"debug": False, "plugin_name": "local"},
-        description="Backend to use in this compute resource set.",
-        discriminator="plugin_name",
-        title="Backend",
-    )
-    scaling_strategy: Optional[
-        Union[MaxAvailableResourceScaling, KubernetesResourceScaling]
-    ] = Field(
-        {"match_all_requirements": False, "plugin_name": "max_available_resource_scaling"},
-        description="Scaling strategy to use in this compute resource set.",
-        discriminator="plugin_name",
-        title="Scaling Strategy",
+        {'debug': False, 'plugin_name': 'local'},
+        description='Backend to use in this compute resource set.',
+        discriminator='plugin_name',
+        title='Backend',
+    )
+    scaling_strategy: Optional[Union[MaxAvailableResourceScaling, KubernetesResourceScaling]] = (
+        Field(
+            {'match_all_requirements': False, 'plugin_name': 'max_available_resource_scaling'},
+            description='Scaling strategy to use in this compute resource set.',
+            discriminator='plugin_name',
+            title='Scaling Strategy',
+        )
     )
     available_resources: Optional[Resources] = Field(
-        {"custom": {}}, description="Available resources in the compute resource set."
+        {'custom': {}}, description='Available resources in the compute resource set.'
     )
     available_applications: Optional[List[ScalerApplicationInfo]] = Field(
-        [], description="List of available applications.", title="Available Applications"
+        [], description='List of available applications.', title='Available Applications'
     )
     evaluator_requirements_matching: Optional[bool] = Field(
         False,
-        description="Whether the evaluators should do matching of resource and software requirements.",
-        title="Evaluator Requirements Matching",
+        description='Whether the evaluators should do matching of resource and software requirements.',
+        title='Evaluator Requirements Matching',
     )
     evaluator_task_directory_cleanup: Optional[EvaluatorTaskDirectoryCleanup] = Field(
-        "always",
-        description="Cleanup policy for task directories that are passed to evaluators.",
-        title="Evaluator Task Directory Cleanup",
+        'always',
+        description='Cleanup policy for task directories that are passed to evaluators.',
+        title='Evaluator Task Directory Cleanup',
     )
     evaluator_auto_shutdown_time: Optional[int] = Field(
         20,
-        description="Time after which to shutdown the evaluator if not running any jobs.",
-        title="Evaluator Auto Shutdown Time",
+        description='Time after which to shutdown the evaluator if not running any jobs.',
```

Comment:
```suggestion
        description='Time after which to shut down the evaluator if not running any jobs.',
```
Shut down is two words as a verb and one word as a noun.